lib/KeyboardAwareHOC.js

```diff
@@ -180,14 +180,13 @@ function listenToKeyboardEvents(ScrollableComponent: React$Component) {
 
     // Keyboard actions
     _updateKeyboardSpace = (frames: Object) => {
-      let keyboardSpace: number =
-        frames.endCoordinates.height + this.props.extraScrollHeight
-      if (this.props.viewIsInsideTabBar) {
-        keyboardSpace -= _KAM_DEFAULT_TAB_BAR_HEIGHT
-      }
-      this.setState({ keyboardSpace })
       // Automatically scroll to focused TextInput
       if (this.props.enableAutoAutomaticScroll) {
```

Comment:
Hey! Can you rename this prop to the intended name? It should be `enableAutomaticScroll`, not `enableAutoAutomaticScroll` (note the extra *`Auto`*). This has been an error I've introduced myself, but it would be great to change it with this PR.